SummaryOfPrimes.java

```diff
@@ -0,0 +1,23 @@
+public class SummaryOfPrimes {
+
+  public static void main(String[] args) {
+    System.out.println(primeSummary());
+  }
+
+  public static int primeSummary() {
+    int primeAmount = 0;
+  for (int i = 1; i < 2000000; i++) {
+    int factors = 0;
+      for (int e = 1; e <= i; e++) {
+        if (i % e == 0) {
+          factors++;
+        }
+      }
+      if (factors < 3) {
+        primeAmount += i;
+        System.out.println(i);
+      }
+    }
+    return primeAmount;
+  }
+}
```

Comment:
Would be worth investigating alternative solutions to see if the execution time can be reduced. Would also suggest a little functional decomposition e.g. break it down so you might have something like a function that generates a list of primes below a given value and have that function find primes by calling another function which would return a boolean value based on whether the value passed in is a prime or not.

Also, minor point but I think this exercise is 'sum of primes' or 'summation of  primes' rather than 'summary of primes'